### ISSUE TITLE
Add project permission denied after step 3

### ISSUE THREAD

**Issue by clash99** [Cadasta/cadasta-platform#307]
After the 3 step "Add new project" I get a permission denied error.  

If in fact I'm not able to add a project, we should hide the button for the user.  Or at the very least not have them go through all 3 steps of the wizard and then be denied :)

http://screencast.com/t/rjyd7lwXr

**Comment by seav** [comment]
Duplicate of  #185?

**Comment by clash99** [comment]
yep - I'll close this one
